cmd/flag-gen/types/types.go

```diff
@@ -16,27 +16,24 @@ limitations under the License.
 
 package types
 
-import (
-	"time"
-)
-
 // go:generate flag-gen -i github.com/caicloud/nirvana/cmd/flag-gen/types -o github.com/caicloud/nirvana/cli
 
+// TODO: why not used?
```

Comment:
flag-gen read this struct to generate code

Reply:
then we have to trick the tool to not warning on this?

Reply:
fixed by adding `// nolint:`